mRemoteV1/Connection/Protocol/RDP/RdpProtocol.cs

```diff
@@ -157,8 +157,26 @@ public override bool Initialize()
                 _rdpClient.AdvancedSettings2.BitmapPeristence = Convert.ToInt32(_connectionInfo.CacheBitmaps);
                 if (_rdpVersion >= Versions.RDC61)
                 {
-                    _rdpClient.AdvancedSettings7.EnableCredSspSupport = _connectionInfo.UseCredSsp;
+                    _rdpClient.AdvancedSettings7.EnableCredSspSupport = _connectionInfo.Protocol == ProtocolType.RDPonVMBus ? true : _connectionInfo.UseCredSsp;
                     _rdpClient.AdvancedSettings8.AudioQualityMode = (uint)_connectionInfo.SoundQuality;
+
+                    if (_connectionInfo.Protocol == ProtocolType.RDPonVMBus)
+                    {
+                        MSTSCLib.IMsRdpExtendedSettings _settings = (MSTSCLib.IMsRdpExtendedSettings)((AxMsRdpClient8NotSafeForScripting)Control).GetOcx();
+                        object value = true;
+                        _settings.set_Property("DisableCredentialsDelegation", ref value);
```

Comment:
Use the `SetExtendedProperty()` class method here instead. It will save you some setup and handle errors gracefully.

Reply:
Fixed by 9128928